tests/test_rebar.py

```diff
@@ -248,6 +248,46 @@ def get_me():
         )
         self.assertEqual(resp.status_code, 400)
 
+    def test_validate_headers_delete(self):
```

Comment:
if you add a docstring comment the testrunner output is going to improve in readability. 
Not a blocking issue though